src/idsk/components/footer-extended/footer-extended.js

```diff
@@ -0,0 +1,241 @@
+import '../../../govuk/vendor/polyfills/Function/prototype/bind'
+import '../../../govuk/vendor/polyfills/Event' // addEventListener and event.target normaliziation
+import { nodeListForEach } from '../../../govuk/common'
+import { toggleClass } from '../../common'
+// import { characterCount } from '../character-count/character-count'
+
+import  CharacterCount  from '../../../govuk/components/character-count/character-count'
+
+
+CharacterCount.prototype.updateCountMessage = function () {
```

Comment:
definovanie override metody nejakej triedu v inej casti inej triedy v tomto dokumente sa nam moze vypomstit v buducnosti, ze nam bude prepisovat nezelane veci.. to skor presun radsej cely komponent do idsk a prepis len tuto metody aspon budeme vediet kde hladat ak nieco nepojde :)